Controller/AuthorizeController.php

```diff
@@ -124,13 +125,12 @@ protected function getRedirectionUrl(UserInterface $user)
     /**
      * @return ClientInterface
      */
-    protected function getClient()
+    protected function getClient(Request $request)
```

Comment:
changing the signature of the protected method is a BC break (and this method is meant to be an extension point). Use the RequestStack instead